### PR TITLE
[FSDE-30615] Fix ua-parser-js Chi documentation dependencies

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**

### CODE REVIEW

**Review by SergioQCL — APPROVED** [review]
PLNT!

**Review by jllr — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    directory: "/src/documentation"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "nuxt"
+        versions: ["3.x"]
+      - dependency-name: "@nuxt/types"
+        versions: ["3.x"]
+      - dependency-name: "copy-webpack-plugin"
+        versions: ["7.x", "8.x", "9.x", "10.x", "11.x"]
```
Comment (glm96) [inline .github/dependabot.yml:13]: You can do [">=7.0.0"]

Comment (glm96) [inline .github/dependabot.yml:13]:
```suggestion
        versions: [">=7.0.0"]
```

Comment (glm96) [reply .github/dependabot.yml:13]: plnt
bmba

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+updates:
+  - package-ecosystem: "npm"
+    directory: "/src/documentation"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "nuxt"
+        versions: ["3.x"]
+      - dependency-name: "@nuxt/types"
+        versions: ["3.x"]
```
Comment (glm96) [inline .github/dependabot.yml:11]:
```suggestion
        versions: [">=3.0.0"]
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/src/documentation"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "nuxt"
+        versions: ["3.x"]
```
Comment (glm96) [inline .github/dependabot.yml:9]:
```suggestion
        versions: [">=3.0.0"]
```

**Comment by glm96** [comment]
nice.

**Pull request merged**